tests/openbis_functions/test_openbis_functs.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+from lebedigital.openbis.expstep import ExpStep
+
+
+@pytest.mark.skip(reason='The directory for the yaml file does not exist in tests workflow. Test is dependent on MinimumWorkingExample')
+def test_load_emodul():
+    """Testing if the metadata from yaml emodul files are imported corretly
+    """
+
+    yaml_path = '../usecases/MinimumWorkingExample/emodul/metadata_yaml_files/Wolf 8.2 Probe 1.yaml'
+
+    test_sample = ExpStep(name='test_sample')
+    test_sample.read_metadata_emodul(yaml_path)
+    test_sample.sync_name('name')
+
+    expected_sample_dict = {'name': 'test_sample', 'type': '', 'metadata': {'diameter': 98.6, 'experimentname': 'Wolf 8.2 Probe 1', 'length': 300.2, 'length_unit': 'mm', 'mix_file': '2014_12_10 Wolf.xls', 'operator_date': '28.01.2015', 'operator_timestamp': '10:56:24', 'remark': 'Kraftgeregelt 3,9 kN/s', 'software_specification':
+                                                                            'MTS793|MPT|DEU|1|2|,|.|:|49|1|1|A', 'specimen_name': 'Wolf 8.2 Probe 1', 'tester_name': 'Kh', 'weight': 5328.0, 'weight_unit': 'g', '$name': 'test_sample'}, 'space': '', 'project': '', 'collection': '', 'parents': [], 'identifier': '', 'permId': '', 'sample_object': None, 'datasets': [], 'dataset_codes': []}
```

Comment:
There should be a way to specify an argumment when running pytest like --noconnection which would skip the tests that would require a human to input the passwords. Would that be something i should look into?

The test is a very basic one. It only checks whether it can load a yaml file as metadata. Writing more useful tests was something i wanted to do later and the tests that i have written were there mostly to test the waters of what would be useful to do. I also did not want to write a lot of mocking tests if there will be a docker image running openbis sometime soon and i can test the connection properly.